### PR TITLE
Updates for Tectonic v1.4 and v1.5 RBAC

### PR DESCRIPTION
**Pull request by alekssaul**
Adds manifests for Tectonic Role Based Access Controls based on Kubernetes 1.4 and 1.5 RBAC specs. 

Fixes: https://github.com/coreos/docs/issues/948

### CODE REVIEW

**Comment by robszumski** [comment]
Does it make more sense to split these into two sections? instead of two stanzas of "this or that"?

**Comment by alekssaul** [comment]
@robszumski : Moved version specific wording into different sections.

File: quay-enterprise/tectonic/index.md [hunk 1]
```diff
@@ -27,6 +27,22 @@ Next, download each of the following files to your workstation, placing them alo
 - [quay-enterprise-service-nodeport.yml](files/quay-enterprise-service-nodeport.yml)
 - [quay-enterprise-service-loadbalancer.yml](files/quay-enterprise-service-loadbalancer.yml)
 
+## Role Based Access Control
+
+Quay Enterprise has native Kubernetes integrations. These integrations require Service Account to have access to Kubernetes API. When Kubernetes RBAC is enabled (Tectonic  v1.4 and later), Role Based Access Control policy manifests also have to be deployed.
+
+Kubernetes API has sligh changes between versions 1.4 and 1.5, Download appropiate versions of Role Based Access Control (RBAC) Policies.
```
Comment (rithujohn191) [inline quay-enterprise/tectonic/index.md]: typo: "sligh"

**Comment by alekssaul** [comment]
Thanks @rithujohn191 

Squashed and Merged

**Pull request merged**